cmd/initialise/sql/cockroach/11_logstore.sql

```diff
@@ -0,0 +1,28 @@
+CREATE SCHEMA IF NOT EXISTS logstore;
```

Comment:
why is it a init job instead of a setup job?

Reply:
Please comment here: https://github.com/zitadel/zitadel/pull/4779?notification_referrer_id=NT_kwDOAMI2IrM0OTQ4MDE4MDM3OjEyNzI3ODQy#issuecomment-1423835243